File: Python/Estrutura condicional/notas.py
# Entrada de dados
nota1 = float(input("Digite a primeira nota: "))
nota2 = float(input("Digite a segunda nota: "))

# Algoritmo matemático
soma = nota1 + nota2

# Condicional
if soma < 60:
    print(f"A sua nota final foi de {soma}, você foi reprovado(a).")
else:
    print(f"A sua nota foi final de {soma}.")

File: C#/Estrutura condicional/notas.cs
// Variáveis
double nota1, nota2, soma;

// Entrada de dados
Console.Write("Digite a primeira nota: ");
nota1 = double.Parse(Console.ReadLine());

Console.Write("Digite a segunda nota: ");
nota2 = double.Parse(Console.ReadLine());

// Algoritmo matemático
soma = nota1 + nota2;

// Condicional
if (soma < 60)
{
    Console.Write("A sua nota foi " + soma + ", portanto você foi reprovado(a).");
}
else
{
    Console.Write("A sua nota foi " + soma);
}